client/src/accountSummary/member/messaging/SendMessage.js

```diff
@@ -6,29 +6,72 @@
  *
  */
 
-import React from 'react';
+import React, {useState} from 'react';
 import PropTypes from "prop-types";
+import {sendMessage} from "../../../services/MessageService"
 import LargeTextArea from "../../../common/forms/LargeTextArea";
 import SubmitButton from "../../../common/forms/SubmitButton";
+import {useEffect} from "react";
+import {checkIfErrorsExistInMapping, validateInput} from "../../../registration/registrationUtils";
 
 function SendMessage(props) {
     //username currently unused, but intended as component which will handle post request to add message to database
-    const {username, otherUser} = props;
-    function onSubmit(){
-        return;
+    const {receiverName, receiverId} = props;
+    const [content, setContent] = useState(undefined);
+    const [contentError, setContentError] = useState(undefined);
+    useEffect(() => {
+        content !== undefined && validateInput(content, setContentError);
+    }, [content]);
+
+    const isFormValid = () => {
+
+        const errors = {
+            content: false,
+        }
+
+        errors.fullDes = validateInput(content, setContentError);
+
+        return !(checkIfErrorsExistInMapping(errors));
     }
+
+    //function for input checks on submit
+    function onSubmitMessage() {
+        if (isFormValid()) {
+            const message = {
+                receiverId: receiverId,
+                content: content
+            }
+            sendMessage(message)
```

Comment:
Optional: everywhere else on the site we follow the format of `MessageService.sendMessage(message)` so that its really easy to see which functions are calls to our API

Reply:
will do

Reply:
addressed